content/en/docs/profiles.md

```diff
@@ -0,0 +1,132 @@
+| [Pending Authorization Lifetime](#pending-authorization-lifetime) | 1 hour |
+| [Authorization Reuse Period](#authorization-reuse-period) | 7 hours |
+| [Order Lifetime](#order-lifetime) | 8 hours |
+| [Certificate Common Name](#certificate-common-name) | No |
+| [Key Encipherment KU](#key-encipherment-key-usage) | No |
+| [TLS Client Auth EKU](#tls-client-authentication-extended-key-usage) | No |
+| [Subject Key ID](#subject-key-identifier-extension) | No |
+| [Validity Period](#validity-period) | 90 days |
+
+</div>
+<div class="boxed">
+
+## shortlived
+
+The shortlived profile is identical to the tlsserver profile, with one key distinction: the resulting certificate is only valid for 6ish days. This allows these certificates to qualify as "Short-Lived Subscriber Certificates" under the Baseline Requirements, which means they do not need to contain any revocation information. This makes the certificates even smaller, and removes any possibility of a client accidentally trusting a certificate after it has been revoked.
+
+We recommend this profile for those who fully trust their automation to renew their certificates on time. This profile is not for everyone. Because this profile results in much higher issuance volume (since certificates need to be renewed every few days, instead of every few months), it is currently locked behind an allowlist. If you would like to be placed on the allowlist, please fill out [this form](TODO TKTK XXX).
+
+| Property | Value |
+| -------- | ----- |
+| [Pending Authorization Lifetime](#pending-authorization-lifetime) | 1 hour |
+| [Authorization Reuse Period](#authorization-reuse-period) | 7 hours |
+| [Order Lifetime](#order-lifetime) | 8 hours |
+| [Certificate Common Name](#certificate-common-name) | No |
+| [Key Encipherment KU](#key-encipherment-key-usage) | No |
+| [TLS Client Auth EKU](#tls-client-authentication-extended-key-usage) | No |
+| [Subject Key ID](#subject-key-identifier-extension) | No |
+| [Validity Period](#validity-period) | 160 hours |
+
+</div>
+
+# Selecting a Profile
+
+The process for selecting a profile is described in [this Internet-Draft](https://datatracker.ietf.org/doc/draft-aaron-acme-profiles/), which we plan to work with the IETF ACME Working Group to turn into a full RFC. Not all ACME Clients have implemented this draft, so the client you use may not yet be able to select a profile.
+
+In general, if you want to select a profile, you should:
+
+1. Read your ACME client's documentation to see if it support profile selection, and if it does, how to tell it which profile you want.
+2. Fetch the Let's Encrypt [production](https://acme-v02.api.letsencrypt.org/directory) or [staging](https://acme-staging-v02.api.letsencrypt.org/directory) directory object to see which profiles are available.
+3. Configure your desired profile within your ACME client.
+
+# Glossary
+
+## Validation Properties
+
+The following validation properties can be controlled by our profiles:
```

Comment:
nit: Ending the line with a colon looks weird now that it's immediately followed by a second-level heading. Maybe something like this?

```suggestion
Below are descriptions of the validation properties that can be controlled by our profiles.
```